package.json

```
@@ -44,7 +44,7 @@
     "@wordpress/eslint-plugin": "^17.13.0",
     "@wordpress/hooks": "3.2.0",
     "@wordpress/i18n": "^4.2.1",
-    "@wordpress/icons": "^5.0.1",
+    "@wordpress/icons": "10.31.0",
```

Do we want this to be a `^`-prefixed dependency? Or should we keep it as an explicit dependency?
```suggestion
    "@wordpress/icons": "^10.31.0",
```

---

Let's set the prefixed version 👍